Check/Database/Fragmentation.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * @file
+ * Contains \SiteAudit\Check\Database\Fragmentation.
+ */
+
+/**
+ * Class SiteAuditCheckDatabaseFragmentation.
+ */
+class SiteAuditCheckDatabaseFragmentation extends SiteAuditCheckAbstract {
+  /**
+   * Implements \SiteAudit\Check\Abstract\getLabel().
+   */
+  public function getLabel() {
+    return dt('Database Fragmentation');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getDescription().
+   */
+  public function getDescription() {
+    return dt("Reports all the tables with fragmentation greater than 5%");
```

Comment:
Single, not double quotes.

`Detect table fragmentation which increases storage space and decreases I/O efficiency.`